src/util/rocksdb/event_listener.rs

```diff
@@ -31,11 +33,26 @@ impl EventListener {
     }
 }
 
+#[inline]
```

Comment:
Can we make this function an impl for the [`ToString`](https://doc.rust-lang.org/std/string/trait.ToString.html) trait?

Reply:
In principle, we need to avoid unnecessary memory allocation. But, this part of code will not become bottleneck. So, whatever😂